src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -701,19 +847,26 @@ internal sealed class TupleFieldSymbol : FieldSymbol
         /// A tuple without backing fields is usable for binding purposes, since we know its name and type,
         /// but caller is supposed to report some kind of error at declaration.
         /// </summary>
-        public TupleFieldSymbol(string name, TupleTypeSymbol containingTuple, TypeSymbol type, FieldSymbol underlyingFieldOpt)
+        public TupleFieldSymbol(string name, TupleTypeSymbol containingTuple, TypeSymbol type, OneOrMany<FieldSymbol> underlyingFieldChainOpt)
```

Comment:
Actually, we don't even need to know the underlying field symbol until lowering, we just keep doing unnecessary work to adjust it during tuple generic substitution. Let's get rid of it.